<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerPercutaneousImageGuidedModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerPercutaneousImageGuidedModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Steps</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="PreRegButton">
           <property name="text">
            <string>Pre-Registration</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RegButton">
           <property name="text">
            <string>Registration</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PostRegButton">
           <property name="text">
            <string>Post-Registration</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PostInsButton">
           <property name="text">
            <string>Post-Insertion</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::West</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="DICOMTab">
         <attribute name="title">
          <string>DICOM</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>AETitle / Port : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="DICOMAETitleBox">
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string>SPLSNR</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="DICOMPortBox">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="inputMask">
               <string>99000; </string>
              </property>
              <property name="text">
               <string>1040</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>DICOM Directory:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="DICOMDirectoryLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="ctkDirectoryButton" name="DICOMListenerButton">
            <property name="text">
             <string>Start DICOM Listener</string>
            </property>
            <property name="options">
             <set>ctkDirectoryButton::HideNameFilterDetails|ctkDirectoryButton::ShowDirsOnly</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DICOMManagerButton">
            <property name="text">
             <string>Open DICOM Manager</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="TargetsTab">
         <attribute name="title">
          <string>Targets</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="qMRMLNodeComboBox" name="TargetListSelector">
              <property name="nodeTypes">
               <stringlist>
                <string>vtkMRMLMarkupsFiducialNode</string>
               </stringlist>
              </property>
              <property name="baseName">
               <string>Targets</string>
              </property>
              <property name="noneEnabled">
               <bool>true</bool>
              </property>
              <property name="renameEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="AddTargetButton">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="RemoveTargetButton">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="TargetListWidget">
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>R</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>A</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>S</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Visibilty</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="ClearTargetListButton">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="WorkspaceTab">
         <attribute name="title">
          <string>Workspace</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="EstimatedRobotPositionButton">
            <property name="text">
             <string>Set Estimated Robot Position</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PlaceRobotButton">
            <property name="text">
             <string>Place Robot</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="RegistrationTab">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <attribute name="title">
          <string>Registration</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Registration Image:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="qMRMLNodeComboBox" name="RegistrationImageSelector"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Robot Fiducial List:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="qMRMLNodeComboBox" name="RegistrationFiducialListSelector"/>
          </item>
          <item>
           <widget class="QPushButton" name="RegisterRobotButton">
            <property name="text">
             <string>Register Robot To Image</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="RobotTab">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <attribute name="title">
          <string>Robot</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Step 1: Zeroing</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Go to Zero</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Step 2: Send Registration</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Send Registration</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Step 3: Send Targets</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QTableWidget" name="tableWidget_2">
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>R</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>A</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>S</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Visibility</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_6">
                 <property name="text">
                  <string>Send Target</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Step 4: Move to Target</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="pushButton_7">
               <property name="text">
                <string>Move</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_8">
               <property name="text">
                <string>Hold</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Emergency</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="pushButton_9">
               <property name="text">
                <string>EMERGENGY STOP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_10">
               <property name="text">
                <string>Reset Robot</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ResliceTab">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <attribute name="title">
          <string>Reslice</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerPercutaneousImageGuidedModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TargetListSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>3</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
